pkg/provisioner/fss/fss.go

```diff
@@ -148,19 +152,20 @@ func (fsp *filesystemProvisioner) Provision(options controller.VolumeOptions, ad
 			},
 		})
 		if err != nil {
-			glog.Errorf("Failed to create a file system options=%#v: %v", options, err)
+
+			fsp.logger.With(zap.Error(err), "options", options).Error("Failed to create a file system options.")
 			return nil, err
 		}
 		fsID = *resp.FileSystem.Id
 	}
 
 	target, err := fsp.getOrCreateMountTarget(ctx, options.Parameters[mntTargetID], *ad.Name, options.Parameters[subnetID])
 	if err != nil {
-		glog.Errorf("Failed to retrieve mount target: %s", err)
+		fsp.logger.With(zap.Error(err)).Error("Failed to retrieve mount target.")
 		return nil, err
 	}
 
-	glog.V(6).Infof("Creating export set")
+	fsp.logger.Info("Creating export set.")
```

Comment:
Re-assign the function-level `logger` to include `"mountTargetID"`, `"exportPath"`, and `"exportSetID"` fields and then `logger.Info("Creating Export")`